elixir/apps/domain/lib/domain/changes/hooks/tokens.ex

```diff
@@ -1,39 +1,19 @@
 defmodule Domain.Changes.Hooks.Tokens do
   @behaviour Domain.Changes.Hooks
-  alias Domain.{Flows, PubSub, Tokens}
+  alias Domain.{PubSub, Tokens}
   import Domain.SchemaHelpers
 
   @impl true
   def on_insert(_lsn, _data), do: :ok
 
   @impl true
-
   # updates for email and relay_group tokens have no side effects
-  def on_update(_lsn, %{"type" => "email"}, _data), do: :ok
-
-  def on_update(_lsn, _old_data, %{"type" => "email"}), do: :ok
-
-  def on_update(_lsn, %{"type" => "relay_group"}, _data), do: :ok
-
-  def on_update(_lsn, _old_data, %{"type" => "relay_group"}), do: :ok
-
-  # Soft-delete - process as delete
-  def on_update(lsn, %{"deleted_at" => nil} = old_data, %{"deleted_at" => deleted_at})
-      when not is_nil(deleted_at) do
-    on_delete(lsn, old_data)
-  end
-
-  # Regular update
```

Comment:
@jamilbk, please let me know if I've overlooked something here, but the only special case appeared to be the soft-delete case, which isn't a thing anymore.